pkgs/development/python-modules/pex/do-not-fetch-windows-script-stubs.patch

```diff
@@ -0,0 +1,26 @@
+--- a/build-backend/pex_build/setuptools/build.py
++++ b/build-backend/pex_build/setuptools/build.py
+@@ -79,23 +79,6 @@ def build_wheel(
+                 (installed_file.path, installed_file) for installed_file in Record.read(fp)
+             )
+ 
+-        for stub in windows.fetch_all_stubs():
```

Comment:
This creates a broken Pex distribution since Pex has always support creating a PEX for a target platform while running from a totally different platform. If a Nix user wants to build a PEX for Windows users, they'll get an ugly error. Not using Nix, but say some Linux distro or macOS straight up, they won't get the error.

Reply:
True - I didn't test building a PEX for different platforms. 

Would that have worked without the stubs with pex 2.29.0 or earlier? Asking because I personally never used that functionality and tests were already disabled when I added myself as a maintainer in nixpkgs. I don't use pex at all anymore and am removing myself as a maintainer here. The other maintainer @copumpkin doesn't seem to be very active on github anymore, so we could either keep it on a best-effort basis or drop it from nixpkgs altogether.

Not sure why @bengsparks opened up an upstream issue when he explicitly does *not* want to maintain this package.